recipes/_install.rb

```diff
@@ -37,6 +37,12 @@ def nsp
   notifies :delete, 'file[splunk-marker]', :immediately
 end
 
+service 'splunk-stop' do
+  service_name service
+  action :nothing
+  supports stop: true, nothing: true
```

Comment:
That was my initial thought. But this was the easiest way to get this done without renaming the 'splunk-start' service.

Now I renamed the 'splunk-start' service to 'splunk' and the old 'splunk' service (used for restart) to 'splunk-restart'. 

b401e22f1ee8a89265e57e19c4ad8c8c5ff6306b